swagger/v1/api_spec.json

```diff
@@ -992,6 +992,64 @@
             }
           }
         }
+      },
+      "post": {
+        "summary": "Submit a NPQ outcome for a single participant",
+        "operationId": "npq_outcome_post",
+        "tags": [
+          "Participants Outcomes"
+        ],
+        "security": [
+          {
+            "bearerAuth": [
+
+            ]
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/NPQOutcomeRequest"
```

Comment:
when attempting to load the API references in v1 and v2 I got the following error:
```
Invalid data for ref: #/components/schemas/NPQOutcomeRequest is not defined
```
Could you add this component to both v1 and v2 please?

Reply:
Hi @cwrw it's been addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2809/commits/5b834ebb8302981007dfb76a0981569bc0920e886 tks for your review.